badge-maker/lib/make-badge.js

```diff
@@ -24,22 +23,6 @@ module.exports = function makeBadge({
   label = `${label}`.trim()
   message = `${message}`.trim()
 
-  // This ought to be the responsibility of the server, not `makeBadge`.
-  if (format === 'json') {
```

Comment:
I haven't read over the code for this bit at all yet. I agree with the objective. Has anything significant has changed since https://github.com/badges/shields/pull/4980 in terms of the implementation, specifically https://github.com/badges/shields/pull/4980#issuecomment-623119680 ? I will have a look through it. If a slightly different configuration of not eating our own dogfood is necessary to get off scoutcamp, this shouldn't be a blocker but until we do #4947 and #4949 I suspect all we can really do is move from one undesirable implementation to another. I do accept #4947 and #4949 have been open for 2 years and none of us are actively working on them.
As a note, we may want to consider documenting it as a non-BC change for badge-maker, even though `format='json'` is undocumented. The next badge-maker will be a major release anyway due to dropping node 10.